content/fr/docs/4.directory-structure/3.components.md

```diff
@@ -122,39 +122,36 @@ Si on a des composants dans des répertoires imbriqués comme ceci:
 components/
   base/
       foo/
-         Button.vue
+         CustomButton.vue
 ```
 
 Nous recommandons d'utiliser le nom du répertoire dans le nom du fichier pour plus de clarté.
 
 ```html
-<BaseFooButton />
+<BaseFooCustomButton />
 ```
 
-Cependant, si l'on souhaite garder `Button.vue`, on peut utiliser l'option `prefix` dans le fichier de configuration `nuxt.config.js` pour ajouter un préfixe à tous les composants dans un répertoire donné.
-
-```bash
-components/
-  base/
-      foo/
-         Button.vue
-```
+Si nous voulons l'utiliser comme `<CustomButton />` tout en gardant la structure de répertoire, nous pouvons ajouter le répertoire de `CustomButton.vue` dans `nuxt.config.js`.
 
 ```bash{}[nuxt.config.js]
 components: {
   dirs: [
     '~/components',
-    '~/components/base'
+    '~/components/base/foo'
   ]
 }
 ```
 
-Ainsi, on pourra utiliser `BaseButton` dans le template or lieu de `Button` sans pour autant devoir changer le nom du fichier `Button.vue`.
+Et maintenant nous pouvons utiliser `<CustomButton />` au lieu de `<BaseFooCustomButton />`.
 
 ```html{}[pages/index.vue]
-<FooButton />
+<CustomButton />
 ```
 
+::alert{type="next"}
+Voir [la propriété components](/docs/configuration-glossary/configuration-components) pour d'autres méthodes de contrôle du nom des composants.
```

Comment:
Same here, I had this part translated by a translation service. I'm not sure if the translation is correct or not, so I would like you to check it for me. 🙏

Reply:
LGTM